_data/toc.yaml

```diff
@@ -1027,18 +975,87 @@ manuals:
       title: Using Docker Cloud and Packet.net
     - path: /docker-cloud/infrastructure/cloud-on-aws-faq/
       title: Using Docker Cloud on AWS
-  - title: Cloud API reference
-    path: /apidocs/docker-cloud/
-  - title: Cloud stack file YAML reference
-    path: /docker-cloud/apps/stack-yaml-reference/
+  - sectiontitle: Standard Node and App Management
```

Comment:
Let's change this title to: "Manage Infrastructure and Applications (Standard-Mode)"